packages/platform/platform-mcp/src/decorators/resource.spec.ts

```
@@ -0,0 +1,33 @@
+import {ContentType, Description, Title} from "@tsed/schema";
+
+import {Resource} from "../..";
+
+@Injectable()
+export class TestResource {
+  @Resource("tsed://init/options")
+  @Title("Title")
+  @Description("Description")
+  @ContentType("application/json")
+  resource() {}
+}
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)